### PR TITLE
Set date delivered

### PR DESCRIPTION
**Pull request by mayabrandi**

### CODE REVIEW

File: EPPs/copy_process_UDF_to_samples.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    def __init__(self, process):
+        self.process = process
+        self.date = process.udf.get('Date delivered')
+        self.samples = []
+
+    def get_samples(self):
+        all_artifacts = self.process.all_inputs()
+        for art in all_artifacts: self.samples += art.samples
+        self.samples = list(set(self.samples))
```
Comment (mayabrandi) [inline EPPs/copy_process_UDF_to_samples.py]: Remove list?

File: EPPs/copy_process_UDF_to_samples.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import sys
+
+DESC = """Script to copy udf from process to samples
+
+Written by Maya Brandi, Science for Life Laboratory, Stockholm, Sweden
+"""
+
+
+class CopyUdfs():
+
```
Comment (mayabrandi) [inline EPPs/copy_process_UDF_to_samples.py]: Less general class name?

File: EPPs/copy_process_UDF_to_samples.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    def __init__(self, process):
+        self.process = process
+        self.date = process.udf.get('Date delivered')
```
Comment (mayabrandi) [inline EPPs/copy_process_UDF_to_samples.py]: Get the date where it is used instead

File: EPPs/copy_process_UDF_to_samples.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    def set_date(self):
+        for sample in self.samples:
+            sample.udf['Delivered'] = self.date
+            sample.put()
+
+def main(lims, args):
+    process = Process(lims, id = args.pid)
+    CUdf = CopyUdfs(process)
+    
+    if not CUdf.date:
```
Comment (mayabrandi) [inline EPPs/copy_process_UDF_to_samples.py]: Handle this from within the class?

File: EPPs/copy_process_UDF_to_samples.py (hunk 2 above)
Comment (mayabrandi) [reply EPPs/copy_process_UDF_to_samples.py]: Ill thet the name of the class stay general. The class is only used from within the script so there is no need for more descriptive class name. Is my opinion :) @sylvinite

**Comment by mayabrandi** [comment]
Im happy with this now @sylvinite What do you think?

**Comment by sylvinite** [comment]
Looks good. Review done in person.

**Pull request merged**